tests/test_keybind.py

```diff
@@ -115,15 +117,17 @@ def test_keys_inactive(hlwm, keyboard, maskmethod, whenbind, refocus):
 
 
 def test_invalid_keys_inactive_via_rule(hlwm, keyboard):
-    hlwm.call('keybind x close')
+    hlwm.call('keybind x add anothertag')
     # Note: In future work, we could make this fail right away. But
     # currently, that is not the case.
     hlwm.call('rule once keys_inactive=[b-a]')
     hlwm.create_client()
```

Comment:
This can be removed now, right?

Reply:
And where would the client with `keys_inactive` be then?

Reply:
You're right, that was a brainfart.